nixos/modules/config/iproute2.nix

```diff
@@ -0,0 +1,20 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.networking.iproute2;
+  runDir = "/run/iproute2";
```

Comment:
Not to nitpick too much, but why not make this a config option like ```nixpkgs.config.iproute2.confdir``` & use it in place of runDir.